src/components/dashboard/GapByRace.vue

```diff
@@ -58,37 +145,67 @@ const activeStats = computed(() => {
   );
   if (row) {
     // don't let a population be more than 100% vaccinated
-    return fieldData.value.map((f) => {
-      const population = row[f.populationField];
-      return {
-        name: f.name,
-        pct:
-          population > 0
-            ? Math.min(1, row[f.observedField] / row[f.populationField])
-            : 0,
-        gap: Math.max(0, row[f.expectedField] - row[f.observedField]),
-        population,
-      };
-    });
+    return fieldData.value
+      .map((f) => {
+        const population = row[f.populationField];
+        return {
+          name: f.name,
+          pct:
+            population > 0
+              ? Math.min(1, row[f.observedField] / row[f.populationField])
+              : NaN,
+          gap: Math.max(0, row[f.expectedField] - row[f.observedField]),
+          population,
+        };
+      })
+      .sort(sortByProperty("pct"));
   } else {
     return [];
   }
 });
 
+// activeStats is sorted by pct, minVaxRace is the first group whose percentage gap is not NaN
 const minVaxRace = computed(() => {
-  let minRow = activeStats.value[0];
-  activeStats.value.forEach((stat) => {
-    if (stat.pct < minRow.pct) {
-      minRow = stat;
+  for (let i = 0; i < activeStats.value.length; i++) {
+    if (!isNaN(activeStats.value[i].pct)) {
+      return activeStats.value[i];
     }
-  });
-  return minRow;
+  }
+
+  // shouldn't reach here, but if it does return the first one
+  return activeStats.value[0];
+});
+
+const activeFocusStats = computed(() => {
+  return activeStats.value.find(
+    (a) => a.name.toUpperCase() === props.focusStat.value.toUpperCase()
+  );
 });
 </script>
 
-<style scoped>
+<style scoped lang="scss">
 .centered {
   display: grid;
   place-content: center;
 }
+
+.visible {
+  visibility: visible;
+}
+
+.hidden {
+  visibility: hidden;
+}
```

Comment:
there's a bulma utility for this: `is-invisible`

do we need the visible one as well as that's the default value I think?

Reply:
I've updated to use `is-invisible` !